app/controllers/application_controller.rb

```diff
@@ -1,3 +1,4 @@
 class ApplicationController < ActionController::Base
```

Comment:
@andela-morekoya Is there a reason why your ApplicationController is inheriting from `Base` and not from `Api`?